mail-client/mutt/patches/mutt-1.12.2.patchset

```diff
@@ -0,0 +1,51 @@
+From 71de919e2ba5b93d8004047e32adc0186241827b Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Zolt=C3=A1n=20Mizsei?= <zmizsei@extrowerk.com>
+Date: Mon, 26 Dec 2016 21:43:34 +0100
+Subject: [PATCH] Haiku patch
+
+---
+ getdomain.c | 4 ++++
+ 1 file changed, 4 insertions(+)
+
+diff --git a/getdomain.c b/getdomain.c
+index 15803af..8abd415 100644
+--- a/getdomain.c
++++ b/getdomain.c
+@@ -28,6 +28,10 @@
+ 
+ #include "mutt.h"
+ 
++#ifdef __HAIKU__
++#define _SC_HOST_NAME_MAX       72
++#endif
++
+ 
+ int getdnsdomainname (char *d, size_t len)
+ {
+-- 
+2.10.2
+
+From 7ca52826e0ea0996631200cfb2ec3d578ac225ee Mon Sep 17 00:00:00 2001
+From: Ken Mays <kmays2000@gmail.com>
+Date: Mon, 23 Sep 2019 18:24:15 +0100
+Subject: [PATCH] Mutt 1.12 EX_OK patch
+
+diff --git a/sendlib.c.older b/sendlib.c
+index 9ec0ccd..8b5752d 100644
+--- a/sendlib.c.older
++++ b/sendlib.c
+@@ -47,11 +47,11 @@
+ #include <sys/wait.h>
+ #include <fcntl.h>
+ 
+-#ifdef HAVE_SYSEXITS_H
+-#include <sysexits.h>
++/* #ifdef HAVE_SYSEXITS_H
++#include <sysexits.h> 
+ #else /* Make sure EX_OK is defined <philiph@pobox.com> */
```

Comment:
So ... you are just going to include a double-open comment here? That is pretty bad form.